docs/src/internals/montecarlo.md

```diff
@@ -163,10 +163,10 @@ If provided, the generated trials and results will be saved in the indicated
 `trials_output_filename` and `results_output_dir` respectively. If `results_in_memory` is set
 to false, then results will be cleared from memory and only stored in the
 `results_output_dir`. After `run`, the results of a `SimulationInstance` can be accessed using
-the familiar `getindex` function with the following signature, which returns a `DataFrame`. 
```

Comment:
Not sure the word "familiar" is still appropriate.

Reply:
Good point I'll change that